dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -831,6 +839,20 @@
         }
       ]
     },
+    "CustomHealthcareTaskParameters": {
+      "type": "object",
+      "description": "Supported parameters for a custom Healthcare task.",
+      "properties": {
+        "stringIndexType": {
+          "$ref": "common.json#/definitions/StringIndexType"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "common.json#/definitions/PreBuiltTaskParameters"
```

Comment:
Are we sure we want to use prebuilt params instead of [custom params](https://github.com/Azure/azure-rest-api-specs/blob/7eda52e0792c584c722016ef5ee533f923ba41cf/dev/cognitiveservices/data-plane/Language/analyzetext.json#L560)?

Reply:
You're right. Thanks for pointing this out.